framework/src/meshgenerators/TiledMeshGenerator.C

```diff
@@ -67,7 +67,7 @@ TiledMeshGenerator::generate()
 {
   std::unique_ptr<MeshBase> initial_mesh = std::move(_input);
   if (!initial_mesh->is_replicated())
-    mooseError("SmoothMeshGenerator is not implemented for distributed meshes");
+    mooseError("TiledMeshGenerator is not implemented for distributed meshes");
```

Comment:
We could just omit the name since it comes out automatically in `MooseObject::mooseError`